.github/workflows/discussion-triage.yml

```diff
@@ -0,0 +1,28 @@
+name: Discussion Triage
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'discuss'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create a discussion in github/cli
+        run: |
+            export TITLE="Triage: $(gh issue view ${ISSUE_NUMBER} --json title --template {{.title}}) (#${ISSUE_NUMBER})"
+            gh api graphql -f query='
+              mutation($title: String!, $body: String!) {
+                createDiscussion(
+                    input: {repositoryId: "R_kgDODAqwpw", categoryId: "DIC_kwDODAqwp84CajJa", title: $title, body: $body}
```

Comment:
I could pull the `repositoryId` out into a variable but I figure it's actually not supposed to change so 🤷 

The `categoryID` is the ID for the `triage` category. I fetched it with:

```
gh api graphql -f query='
{
  # https://github.com/github/docs

  repository(owner: "github", name: "cli") {
    id # RepositoryID
    name
    discussionCategories(first: 10) {
      nodes {
        id # CategoryID
        name
      }
    }
  }
}'
```